tests/smoke/base-template.sh

```diff
@@ -54,3 +54,9 @@ cookiecutter --no-input \
 echo "${OUTPUT_DIR}"
 
 set +ex
+
+if [[ "${input_params}" == *"use_circleci=yes"* ]] && command -v circleci &> /dev/null; then
```

Comment:
this block should be moved to tests/smoke/ci.sh

you don't need to have this `if` .. I can share an idea for this test in the ci.sh